lib/service_sign_in/file_self_assessment.en.yaml

```diff
@@ -3,16 +3,17 @@ locale: en
 choose_sign_in:
   title: Prove your identity to continue
   slug: prove-identity
-  description: If you’re filing online for the first time, you need to register for Self Assessment first.
+  description:
```

Comment:
`description` isn't a required field so we can remove this line from the file (otherwise it breaks if we try to import a blank value 😊 )